anime_downloader/commands/config.py

```diff
@@ -0,0 +1,120 @@
+import json
+import os
+import sys
+import click
+import time
+import logging
+from tabulate import tabulate
+from anime_downloader.sites import ALL_ANIME_SITES
+sitenames = [v[1] for v in ALL_ANIME_SITES]
+def clear():    
+    os.system('cls||clear')
+def get_config(): #can't import config directly because of circular import
+    APP_NAME = 'anime downloader'
+    with open(os.path.join(click.get_app_dir(APP_NAME), 'config.json')) as json_file:
+        data = json.load(json_file)
+    return data
+
+
+def create_table(values, option = "setting"):
+    if values == str:
+        val = click.prompt(f'Input the choice here or choose {option} (0 to go back): ', type=str, default="0")
+        clear()
+        return None if val == "0" else val
+    else:
+        table = [[a+1,[b for b in values][a]] for a in range(len(values))]
+        clear()
+        table = tabulate(table, tablefmt='psql')
+        table = '\n'.join(table.split('\n')[::-1])
+        click.echo(table)
+        val = click.prompt(f'Choose {option} (0 to go back): ', type=int, default=1, err=True)
+        return None if val == 0 else [a for a in values][val-1] 
+
+
+#If you are having trouble to understand how this works then take a look at the link below
+#https://stackoverflow.com/questions/13687924/setting-a-value-in-a-nested-python-dictionary-given-a-list-of-indices-and-value
+def nested_set(dictionary, keys, value, create_missing=False): 
+    d = dictionary
+    for key in keys[:-1]:
+        if key in d:
+            d = d[key]
+        elif create_missing:
+            d = d.setdefault(key, {})
+        else:
+            return dic
+    if keys[-1] in d or create_missing:
+        d[keys[-1]] = value
+    return dictionary
+
+layout = {
```

Comment:
I don't like copying over the config. You have to import the DEFAULT_SETTINGS from config.py